apps/meteor/app/lib/server/lib/sendNotificationsOnMessage.js

```diff
@@ -372,8 +372,11 @@ export async function sendAllNotifications(message, room) {
 			}),
 		)
 			.then((users) => {
+				const subscriptions = Subscriptions.findByRoomIdAndUserIds(room._id, users, {
+					fields: { 'u._id': 1 },
+				});
```

Comment:
you need all fields here
```suggestion
				const subscriptions = Subscriptions.findByRoomIdAndUserIds(room._id, users);
```